src/main.ts

```diff
@@ -11,6 +11,7 @@ const routerHistory = createWebHashHistory();
 const router = createRouter({
     history: routerHistory,
     routes: [
+        { path: '/nearSuccess', component: Swap, props : {nearSuccess: 'true'}},
```

Comment:
Maybe we could use something like this:
```
{
    path: '/nearSuccess', redirect: (to) => {
        store.dispatch('account/connect', 'NearConnector');
        return '/';
    },
},
```
That way we don't the prop and is probably safer on component re-renders. What do you think?

Reply:
That's really clean, thanks for the suggestion, will keep a solution like this in mind for the future. Fixed in latest amend.